demo/load/index.html

```diff
@@ -17,28 +17,31 @@
     <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
     <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="../../web-animations.html">
     <link rel="import" href="../../../iron-flex-layout/iron-flex-layout.html">
     <link rel="import" href="../../neon-animated-pages.html">
     <link rel="import" href="../../neon-animations.html">
     <link rel="import" href="full-page.html">
 
-    <style is="custom-style">
-      body {
-        overflow: hidden;
-        @apply(--layout-fullbleed);
-      }
-      full-page {
-        @apply(--layout-fit);
-      }
-    </style>
+    <custom-style>
+      <style is="custom-style">
+        body {
+          overflow: hidden;
+          @apply --layout-fullbleed;
+        }
+        full-page {
+          @apply --layout-fit;
+        }
+      </style>
+    </custom-style>
   </head>
   <body>
 
     <full-page></full-page>
 
     <script>
 
-      document.addEventListener('WebComponentsReady', function() {
+      window.addEventListener('WebComponentsReady', function() {
```

Comment:
nit: this doesn't need to change, it was fixed in webcomponents rc.8 which now fires the event from document